starters/angular-ngrx-scss/src/app/fetch-example/loader/loader.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="loading">
+  <div class="loading__section">
+    <div class="loading__wrapper">
+      <div class="loading__item" [style.height]="height" [style.width]="width">
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
This DOM structure can be simplified greatly with the stylesheet changes I mention elsewhere in this review.

```suggestion
<div class="loader"></div>
```